components/map/MapBackground.tsx

```diff
@@ -32,6 +111,66 @@ const styles = StyleSheet.create({
     width: '100%',
     height: '125%',
   },
-});
+}); 
+//renvoi les informations des moyens lié à l'intervention en cours
+async function MeansLoading() {
+  const {data , error}= await supabase.from(Tables.INTERVENTIONS_MEANS_LINK).select('mean_id,'+Tables.MEANS+'(id)').eq(Columns.INTERVENTION_ID,usePathname().split('/')[2])
```

Comment:
Utiliser des template strings plutot que de la concatenation dégueu

Ça devrait ressembler à un truc comme ça
```suggestion
  const {data , error}= await supabase.from(Tables.INTERVENTIONS_MEANS_LINK).select(`mean_id, ${Tables.MEANS} (id)`).eq(Columns.INTERVENTION_ID,usePathname().split('/')[2])
```